fraqbot/Local/coins.py

```diff
@@ -258,10 +258,23 @@ def _get_all_balances(self):
         
         return f'```{out}```'
 
+    def _get_rank(self, user_id):
```

Comment:
Overall this is headed in the right direction, I think. I'm wondering if it can be more efficient though. You're looping through all the data twice, PLUS a sort. HMU on slack if you want to discuss.